src/views/Order.vue

```diff
@@ -27,9 +27,13 @@
           </div>
 
           <!-- TODO: Update timeline to display the orderCreatedDate, orderCompletedDate, brokeredDate and entryDate -->
-          <!-- <div class="timeline">
-            <ion-item lines="none">
-              <ion-icon slot="start" :icon="timeOutline" class="mobile-only" />
+          <div class="timeline">
```

Comment:
Use Timeline component here

Reply:
I have created a timeline detail component and used it on the order, purchase order, shipment, and timeline page.